src/reference-manual/types.Rmd

```diff
@@ -1386,6 +1386,43 @@ As with other assignments, promotions will happen if necessary (of
 `int` to `real` and of `real` to `complex`, along with the
 corresponding container type promotions).
 
+### Unpacking assignment of tuples {-}
+
+For convience of using values stored in tuples, Stan supports
+"unpacking" (or "destructuring") of tuples in an assignment statement.
+
+Given a tuple `T` of type `tuple(T1, ..., Tn)` and a sequence of
+variables of types `v1`, ..., `vn`, where each `vi` has a type which is
+[assignable](#assignment-typing) from type `Ti`, individual elements of
+the tuple may be assigned to the corresponding variables in the sequence
+by the statement
+
+```stan
+(v1, /*...*/, vn) = T;
+```
+
+For example, if `T` is a tuple of type `tuple(int, real, complex)`,
```

Comment:
[optional]
I would use lower case `t` or `x` for the tuple here to avoid confusion with `T` being read as a generic type.